pkg/config/config_template.yaml

```diff
@@ -876,6 +876,24 @@ api_key:
   #   - 'sql*'
   #   - '*pass*d*'
 
+{{- if .Profiling -}}
+  ## @param profiling - custom object - optional
+  ## Enter specific configurations for profiling.
+  ## Uncomment this parameter and the one below to enable profiling.
+  ## See https://docs.datadoghq.com/tracing/profiling/
+  #
+  ## NOTE: If enabled this option may incur billing charges or other
```

Comment:
```suggestion
  ## NOTE: If enabled this option may impact your billing, or enable
```

Reply:
@sarina-dd I updated this, can you take another look please?